pytest_mh/utils/tc.py

```diff
@@ -0,0 +1,54 @@
+from __future__ import annotations
+
+from .. import MultihostHost, MultihostUtility
+from ..ssh import SSHLog, SSHProcessError
+
+__all__ = ["LinuxTrafficControl"]
+
+
+class LinuxTrafficControl(MultihostUtility):
+    """
+    Perform traffic control operations on remote host.
+
+    All changes are automatically reverted when a test is finished.
+    """
+
+    def __init__(self, host: MultihostHost) -> None:
+        """
+        :param host: Remote host instance.
+        :type host: MultihostHost
+        """
+        super().__init__(host)
+        self.__restore = set()  # type: set[str]
+
+        # Let's find out the interface that connects the systems
+        self.__interface: str = ""
+        result = self.host.ssh.run(f"dig +short {host.hostname}", log_level=SSHLog.Error)
+        try:
+            result = self.host.ssh.run(f"ip -o address | grep {result.stdout}", log_level=SSHLog.Error)
+            self.__interface = result.stdout.split(" ")[1]
+        except SSHProcessError:
+            print(f"{host.hostname} doesn't provide 'ip' command")
```

Comment:
...and what if the  command is present but fails?

I would avoid this, in the end, if a command is not available it will be visible in the stderr log of the result.

Also you can't use print, you need to use self.logger to display messages.

Reply:
Ops! Sorry for the `print()` that was unintended. Can we have an empty `except` in python? Maybe using `pass`?

Reply:
What about this one? Should I remove the `try` or keep it as it is now?

My knowledge of Python is low and I'm trying to take advantage of what I know for Java exceptions. There if an exception is thrown you are forced to catch it, but I don't know if the same applies in Python.

Reply:
Sorry, I just read the next comment. Disregard the previous message.